microservices/nris_api/backend/app/commands.py

```diff
@@ -1,15 +1,29 @@
 import click
 
-from app.extensions import db, oracle_db
+from app.extensions import db, sched
+from app.nris.models.nris_raw_data import NRISRawData
+from app.nris.etl.nris_etl import import_nris_xml, etl_nris_data, clean_nris_data, clean_nris_xml_import
+from app.nris.scheduled_jobs import nris_jobs
 
 
 def register_commands(app):
     @app.cli.command()
-    def _test_oracle_db():
-        cursor = oracle_db.cursor()
+    def _get_nris_raw_data():
```

Comment:
Do these actually give you data on the console somewhere? The naming suggests im getting the data as output, but im assuming im really just importing the data into the database right?

Reply:
It imports the data into the database. Get to me doesn't suggest console output, but I prefer import either way so I'll update the name.

Reply:
I agree with Gyan on this one - especially if you're having it as a cli command. Import is a little more clear I think.

Reply:
Renamed.